### PR TITLE
Remove mention to examples folder

### PR DESCRIPTION
**Pull request by clausmichele**
Fixes https://github.com/Open-EO/openeo-processes/issues/513

### CODE REVIEW

**Comment by clausmichele** [comment]
@m-mohr can we merge this simple fix?

**Comment by m-mohr** [comment]
I think we had a PSC decision on this and I hadn't had the time yet to check what the actual outcome was. 
I just want to update it directly in the right way, not merge yet another potentially wrong thing.

File: README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ This repository contains a set of files formally describing the openEO Processes
 ## Process
 
 * All new processes must be added to the [`proposals`](proposals/) folder.
-* Processes will only be moved from proposals to the stable process specifications once there are at least two implementations and an example process in the [`examples`](examples/) folder showing it in a use case. This doesn't require a PSC vote individually as it's not a breaking change, just an addition.
+* Processes will only be moved from proposals to the stable process specifications once there are at least two implementations. This doesn't require a PSC vote individually as it's not a breaking change, just an addition.
```
Comment (m-mohr) [inline README.md:43]: We never explicitly discussed to remove this requirement (which we could discuss due to the fact that openeo-community-examples is pretty unmaintained), but we did discuss moving the examples:
https://github.com/Open-EO/PSC/issues/14

So the correct way currently is to change as follows unless another PSC discussion takes place:
```suggestion
* Processes will only be moved from proposals to the stable process specifications once there are at least two implementations and an example process in the [`openEO community examples`](https://github.com/Open-EO/openeo-community-examples/) showing it in a use case. This doesn't require a PSC vote individually as it's not a breaking change, just an addition.
```

Comment (clausmichele) [reply README.md]: Fine, thanks for the feedback

**Comment by soxofaan** [comment]
not sure if intended, but I noticed this PR was applied to master instead of going through draft

**Pull request merged**